Reference/Cache/index-v9.md

```diff
@@ -2,9 +2,91 @@
 versionFrom: 9.0.0
 ---
 
-# Caching
+# Cache & Distributed Cache
+
+_This section refers to how to implement caching features in the Umbraco application in a consistent way that will work in both single server environments and load balanced (multi-server) environments. The caching described in this section relates to application caching in the context of a web application only._
+
+## IF YOU ARE CACHING, PLEASE READ THIS
+
+Although caching is a pretty standard concept it is very important to make sure that caching is done correctly and consistently. It is always best to ensure performance is at its best before applying any cache and also beware of *over caching* as this can cause degraded performance in your application because of cache turnover.
+
+In normal environments caching seems to be a pretty standard concept. If you are a package developer or developer who is going to publish a codebase to a loadbalanced environment then you need to be aware of how to invalidate your cache properly, so that it works in load balanced environments. If it is not done correctly then your package and/or codebase will not work the way that you would expect in a load balanced scenario.
```

Comment:
I think this section should be added as a warning that way it is more clear to people that they should be aware of the points you are making.

Reply:
Feel free to do such changes.. This is just updated from v7 as close to 1-1 as possible, while still being valid for v9.

Reply:
Ahhh sorry, I have made the change now.